tokio/src/process/mod.rs

```diff
@@ -1212,6 +1212,24 @@ pub struct Child {
     pub stderr: Option<ChildStderr>,
 }
 
+impl TryFrom<StdChild> for Child {
+    type Error = io::Error;
+
+    fn try_from(value: StdChild) -> io::Result<Self> {
+        let spawned_child = imp::build_child(value)?;
+
+        Ok(Child {
+            child: FusedChild::Child(ChildDropGuard {
+                inner: spawned_child.child,
+                kill_on_drop: false,
+            }),
+            stdin: spawned_child.stdin.map(|inner| ChildStdin { inner }),
+            stdout: spawned_child.stdout.map(|inner| ChildStdout { inner }),
+            stderr: spawned_child.stderr.map(|inner| ChildStderr { inner }),
+        })
```

Comment:
Rather than duplicate all this I'd prefer if we turned this into a call to `Command::build_child` so that these implementations never drift

Reply:
Whoops we can't call `Command::build_child` because it takes `self`, but effectively we should DRY these with a common helper